manifest/kubernetes/rhel/ncp-rhel.yaml

```diff
@@ -950,27 +960,6 @@ spec:
           - name: host-var-run
             mountPath: /var/run/openvswitch
             subPath: openvswitch
-{{if .UseNsxOvsKmod}}
```

Comment:
Please talk with @2ez4szliu if this is fine to be removed, not sure if he already added this part in NCP yaml, so Operator could sync it automatically.

Reply:
ok, checked with latest NCP master, this part has not been added.

Reply:
I think we still need to keep all sections of .UseNsxOvsKmod since they are still needed in operator manifest. 
once shizhao fix in NCP side, we will have same code here, just generated code not handy-input code.

Reply:
Got it, this part has been added.